5.5/root/usr/share/container-scripts/mysql/common.sh

```diff
@@ -114,9 +133,9 @@ function server_id() {
 
 function wait_for_mysql_master() {
   while true; do
-    echo "Waiting for MySQL master (${MYSQL_MASTER_SERVICE_NAME}) to accept connections ..."
+    log_info "Waiting for MySQL master (${MYSQL_MASTER_SERVICE_NAME}) to accept connections ..."
     mysqladmin --host=${MYSQL_MASTER_SERVICE_NAME} --user="${MYSQL_MASTER_USER}" \
-      --password="${MYSQL_MASTER_PASSWORD}" ping &>/dev/null && return 0
+      --password="${MYSQL_MASTER_PASSWORD}" ping &>/dev/null && log_info "MySQL master is ready" && return 0
```

Comment:
is there a way in mysql to assure that master is really ready for accepting connections? this is unrelated, but can we setup readiness check in template to execute a script that will tell us that the "master" is ready?